### ISSUE TITLE
How to enforce a single data cursor in a figure with several plots

### ISSUE THREAD

**Issue by jondo** [joferkington/mpldatacursor#34]
The following code (derived from your `examples/basic.py`) reproduces a problem I see in my current project: It gives me one data cursor per plotted line, but I would like to have only one data cursor in the whole figure. Changing the `display` option does not help.

```python
import matplotlib.pyplot as plt
import numpy as np
from mpldatacursor import datacursor

fig, ax = plt.subplots()
ax.set_title('Click somewhere on a line')

for line in range(1, 5):
    data = np.arange(10) * line
    
    lines = ax.plot(data)

    datacursor(lines)
#    datacursor(lines, display='single')
#    datacursor(lines, display='one-per-axes')

plt.show()
```

I also unseccessfully tried to create one common datacursor for all axes:
```python
import matplotlib.pyplot as plt
import numpy as np
from mpldatacursor import datacursor

fig, ax = plt.subplots()
ax.set_title('Click somewhere on a line')

axe_list = []

for line in range(1, 5):
    data = np.arange(10) * line
    
    lines = ax.plot(data)
    axe_list.append(lines)

datacursor(axes=axe_list)

plt.show()
```

**Comment by jondo** [comment]
Ok, the 2nd try was nonsense, `lines` are not axes. Bear with me, I am quite new to matplotlib.

**Comment by joferkington** [comment]
@anntzer recently fixed this bug in #63. My apologies for not realizing what you were asking!

As far as I can tell, this issue is the same one you were running into, so I'm going to go ahead and close this.

Thanks again for reporting it!

**Comment by jondo** [comment]
Thank you for informing me! I will test at a later time.
